webrtc.html

```diff
@@ -3017,14 +3017,14 @@
         <h4>RTCSessionDescriptionCallback</h4>
         <div>
           <pre class="idl">
-          callback RTCSessionDescriptionCallback = void (RTCSessionDescription sdp);</pre>
+          callback RTCSessionDescriptionCallback = void (RTCSessionDescriptionInit description);</pre>
```

Comment:
Oh, there are two commits, I assumed the description was the commit message. Never mind then, as long as it's landed as two commits or one can otherwise figure out from the resulting commit why this was done.